config/deploy/docker/deploy

```diff
@@ -8,9 +8,12 @@ args = CommandArgs.new
 args.deployments.each do |deployment|
   puts "Deploying #{deployment[:target_group_name]}..."
   Deployer.new(**deployment).run!
-end
 
-EcsTools.new.poll_state_until_stable!(CommandArgs.cluster)
+  # poll between deployments to prevent scaling out too much while we have
+  # double the web containers.
+  # i.e. don't proceed until it has stabilized
+  EcsTools.new.poll_state_until_stable!(CommandArgs.cluster)
```

Comment:
Does this mean it will deploy one app, then wait for it to stabilize, then deploy the next, etc.?

Reply:
Yes. I have a feeling not waiting is related to those times deploys wouldn't go through. We can pull this out if you want.

Reply:
Let's give it a try.  I'm all for waiting a bit if it means not having to spin up a ton of EC2 instances.